vxfreeswitch/voice.py

```diff
@@ -293,6 +355,13 @@ def handle_outbound_message(self, message):
 
         client_addr = message['to_addr']
         client = self._clients.get(client_addr)
+
+        if (client is None and message.get('session_event') ==
+                TransportUserMessage.SESSION_NEW):
+            client = yield self.create_voice_client()
+            yield client.make_call(message['to_addr'])
+            self.register_client(client)
```

Comment:
I think we should follow the pattern used in the server and call ``register_client`` from ``connectionMade`` so that register/deregister both happen from the same class (and we should definitely not yield on anything before registering the client otherwise it might receive messages or get disconnected before we have a change to register it).